srcpkgs/linux5.4/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'linux5.4'
 pkgname=linux5.4
-version=5.4.38
+version=5.4.39
 revision=1
 wrksrc="linux-${version}"
 short_desc="Linux kernel and modules (${version%.*} series)"
 maintainer="Helmut Pozimski <helmut@pozimski.eu>"
 license="GPL-2.0-only"
 homepage="https://www.kernel.org"
 distfiles="https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-${version}.tar.xz"
-checksum=e72b9fa4e3a222d42d64dce9915e5b18f1b75ae722d529a1e560ced2e5d0f30e
+checksum=5141965a61a7b1c538443782ec3ef25fe0c3f9cd4d70c4e9f969d16883f365c8
 python_version=2 #unverified
```

Comment:
Please fix the `xlint` error about unverified `python_version` by setting `python_version=3` and removing the `#unverified` comment thereafter. I believe Linux 5.4 uses Python scripts compatible with Py3.

Reply:
indeed, it compiles fine with `python_version=3`